lib/file/index.js

```diff
@@ -110,7 +110,7 @@ File.prototype.generateBatches = function(done1) {
 		totalCredit += batch.control.totalCredit.value;
         
 		async.each(batch._entries, function(entry, done3) {
-			entry.fields.traceNumber.value = ( entry.fields.traceNumber.value) ?  entry.fields.traceNumber.value : self.header.immediateDestination.value.slice(0,8) + utils.pad(addendaCount, 7, false, '0');
+			entry.fields.traceNumber.value = ( entry.fields.traceNumber.value) ?  entry.fields.traceNumber.value : self.header.immediateOrigin.value.slice(0,8) + utils.pad(addendaCount, 7, false, '0');
```

Comment:
According to the ACH files guidelines:
> The first eight positions of the Trace Number in an Entry Detail Record are the same as the ODFI Routing Number in the corresponding (immediately preceding) Company/Batch Header Record.

@glenselle As far as we could test, the value here must be the one in `immediateOrigin`, or else the file submission fails.